### PR TITLE
Integration of M-Elo

### PR DESCRIPTION
**Pull request by Nimeggis**
## Checklist for reviewer

- The code is easy to understand
- The code follows
  the [coding guidelines](https://github.com/MEITREX/wiki/blob/main/dev-manuals/backend/coding-guidelines.md) of
  this project
- The code is tested or if not, the reason is documented or discussed
- The added and existing tests reasonably cover the code change
- The code has no breaking changes in the database schema or if so, the assignee is aware of it and
  will perform a database migration

### CODE REVIEW

**Review by PaulBredl — DISMISSED** [review]
Wie schon in Discord geschrieben, einmal formatting durchlaufen lassen wäre nciht schlecht

File: src/main/resources/graphql/service/mutation.graphqls [hunk 1]
```diff
@@ -140,6 +164,10 @@ input CreateQuizInput {
 }
 
 input CreateMultipleChoiceQuestionInput {
+    """
+    UUID of the question to update and the id of the corresponding item.
+    """
+    itemId: UUID
```
Comment (myluki2000) [inline src/main/resources/graphql/service/mutation.graphqls:170]: This should most likely not require passing an ID.

File: src/main/resources/graphql/service/mutation.graphqls [hunk 2]
```diff
@@ -372,6 +420,10 @@ input UpdateExactAnswerQuestionInput {
 }
 
 input CreateNumericQuestionInput {
+    """
+    id of the corresponding item
+    """
+    itemId:UUID
```
Comment (myluki2000) [inline src/main/resources/graphql/service/mutation.graphqls:426]: This should most likely not  require passing an ID.

File: src/main/resources/graphql/service/mutation.graphqls [hunk 3]
```diff
@@ -427,6 +479,10 @@ input UpdateNumericQuestionInput {
 }
 
 input CreateSelfAssessmentQuestionInput {
+    """
+    id of the corresponding item
+    """
+    itemId:UUID
```
Comment (myluki2000) [inline src/main/resources/graphql/service/mutation.graphqls:485]: This should most likely not require passing an ID.

**Review by myluki2000 — CHANGES_REQUESTED** [review]
See my longer comments in the gateway PR

**Review by myluki2000 — APPROVED** [review]

**Review by myluki2000 — APPROVED** [review]

**Pull request merged**